src/utils/CvoPoint.cpp

```diff
@@ -29,10 +29,10 @@ namespace cvo {
 //        features_(i, j) = pc[i].features[j];
           point.features[j] = pc[i].features[j];
       }
-      //TODO:fix label
-//      for (int j = 0 ; j < NUM_CLASSES; j++) {
-//        point.label_distribution[j] = pc[i].label_distribution[j];
-//      }
+
```

Comment:
To make sure, can we move all functions related to pcl into a new separate cpp file? The reason is that we will want to allow users to decide if they want to compile with pcl themselves.